scheduling/splitting.go

```diff
@@ -36,7 +36,7 @@ func printResults(courses []structs.Course)(){
 func Split(courses []structs.Course) ([]structs.Course){
     for i, course := range courses {
 		numSections :=  course.NumSections
-		if numSections == 1 && course.CourseCapacity > 200 {
+		if numSections == 0 && course.CourseCapacity > 200 {
 			splitInto := uint(course.CourseCapacity / 100)
```

Comment:
Just wanted to confirm that we wanted to split into sections of 100 after checking if it was over 200 or if it was supposed to be 200